src/test/java/com/google/cloud/tools/appengine/cloudsdk/PathResolverTest.java

```diff
@@ -81,7 +80,7 @@ public void testGetLocationFromLink_valid() throws IOException {
     PathResolver.getLocationsFromLink(possiblePaths, gcloud);
 
     Assert.assertEquals(1, possiblePaths.size());
-   if (SystemUtils.IS_OS_MAC_OSX) {
+   if (System.getProperty("os.name").startsWith("Mac")) {
```

Comment:
Think my comment got lost, but we should just be able to resolve the real path of the sdkHome dir and check that instead of doing a mac specific thing?